simple_classifier/Classifiers/ClassifierProfiler.py

```diff
@@ -0,0 +1,13 @@
+class ClassifierProfiler:
+    def __init__(self, classifiers, profilers, display):
+        self.classifiers = classifiers
+        self.profilers = profilers
+        self.display = display
+
+    def train(self):
+        # Train all classifiers on dataset
+        pass
+
+    def profile_classifiers(self):
+        # Run all profilers on all classifiers and display results
+        pass
```

Comment:
```suggestion
        pass
        
    def display_results(self):
        pass
```

Reply:
Are we adding another method? Just confirming since it isn't in the UML Diagram

Reply:
Sorry, I just came up with it. It essentially just calls `Display` attribute's methods. You can leave it out if you want.

Reply:
Added